ext/intl/grapheme/grapheme_string.c

```diff
@@ -573,8 +553,7 @@ static void strstr_common_handler(INTERNAL_FUNCTION_PARAMETERS, int f_ignore_cas
 	}
 
 	if (needle_len == 0) {
-		zend_argument_value_error(2, "cannot be empty");
-		RETURN_THROWS();
+		RETURN_STRINGL(haystack, part ? 0 : haystack_len);
```

Comment:
Is it necessary to handle this explicitly? It would be better to fall through to avoid behavioral discrepancy wrt UTF-8 validation, for example.